test/dub_package/avg.d

```diff
@@ -59,7 +59,10 @@ void main()
 
     auto id = Identifier.idPool(fname);
     auto m = new ASTBase.Module(&(fname.dup)[0], id, false, false);
-    auto input = readText(fname);
+    auto file = File(fname, "rb");
+    // Create a buffer with four '\0' past the end for lexer
+    auto input = cast(char[]) new ubyte[file.size() + 4];
```

Comment:
32 bit failure in semaphore: 
```
avg.d(64,41): Error: cannot implicitly convert expression `file.size() + 4LU` of type `ulong` to `uint`
```

not sure why the other 32bit CIs don't catch it.

Reply:
Maybe, this is not being tested on other CIs, but I think it should. The value should be clamped to the size_t max value or safely cast to avoid integer overflow and possibly throw an error or assert when it happens. Similar to the behaviour of `read` https://github.com/dlang/phobos/blob/master/std/file.d#L377

That said, is there any strong reason to do this substitution?